tests/unit/testUtapiClient.js

```diff
@@ -240,9 +240,16 @@ describe('UtapiClient:: push metrics', () => {
     });
 
     it('should push abortMultipartUpload metrics', done => {
-        const expected = getObject(timestamp,
-            { action: 'AbortMultipartUpload' });
-        testMetric('abortMultipartUpload', metricTypes, expected, done);
+        const expected = getObject(timestamp, {
+            action: 'AbortMultipartUpload',
+            storageUtilized: '0',
+        });
+        Object.assign(params, metricTypes, { byteLength: 1024 });
+        // Set mock data of one, 1024 byte part object for
+        // `AbortMultipartUpload` to update.
```

Comment:
so you are setting the storageUtilized at 1024 and then subtracting it with the params argument with abortMultipartUpload? Just clarifying.